images/runtime/php/7.2/docker-php-ext-enable

```diff
@@ -0,0 +1,114 @@
+#!/bin/sh
+set -e
+
+extDir="$(php -d 'display_errors=stderr' -r 'echo ini_get("extension_dir");')"
```

Comment:
same question here .. as per my experience I have seen sometimes these scripts doesn't work, so it will be nice if we can test them as well, either manually , better if automated

Reply:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3212379